product-scenarios/1-manage-users-roles/1.1-user-registration/1.1.2-user-registration-by-admin/1.1.2.1-user-registration-through-apis/1.1.2.1.2-scim-2.0/src/test/java/org/wso2/identity/scenarios/test/scim2/ProvisionUserWithValidationFailuresTestCase.java

```diff
@@ -31,52 +29,56 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 import org.wso2.identity.scenarios.commons.ScenarioTestBase;
+import org.wso2.identity.scenarios.commons.util.SCIMProvisioningUtil;
+import org.wso2.identity.scenarios.commons.util.Constants;
 
 import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertNotNull;
 import static org.testng.Assert.assertTrue;
 
+
 public class ProvisionUserWithValidationFailuresTestCase extends ScenarioTestBase {
 
     private CloseableHttpClient client;
     private String PASSWORD = "ab";
+    private String scimUsersEndpoint;
+    private final String SEPERATOR = "/";
+    JSONArray schemasArray;
+
+    HttpResponse response;
 
     @BeforeClass(alwaysRun = true)
     public void testInit() throws Exception {
 
         setKeyStoreProperties();
         client = HttpClients.createDefault();
+        super.init();
     }
 
-    @Test(description = "1.1.1.1.3")
+    @Test(description = "1.1.2.1.2.3")
```

Comment:
Since this value, is used by test grid to identify the test case, better to document on how to use a value for description, for future developers who will start writing tetss cases.

Reply:
the documentation will be updated